jerry-main/cli.c

```diff
@@ -362,7 +362,7 @@ cli_print_help (const char *help) /**< the help message to print */
     cli_print_prefix (help, length);
 
     help += length;
-    while (*help == ' ' && *help != 0)
+    while (*help == ' ')
```

Comment:
If `*help == ' '` then it cannot be 0, since space is 32, not 0.

Reply:
Yes, that's what was spotted by cppcheck and is fixed now.